db/activities/ml_start_here.md

```diff
@@ -12,6 +12,8 @@ presenterAbout: Josiah has interned at Facebook, Microsoft and the CIA. He is cu
 presenterSocials:
   - type: Josiah's Medium
     link: https://medium.com/@josiahcoad
+slackChannel: ml-start-here
+slackChannelLink: "#ml-start-here-slack-link"
```

Comment:
The example usage should be added to `data_science_101.md` and `activity.md`. When we merge, this will go directly to production and we don't have the final names for the channels, so we should probably remove it for now.